packages/now-routing-utils/src/superstatic.ts

```diff
@@ -172,19 +172,13 @@ function replaceSegments(
       }
     }
 
-    for (const [name, value] of Object.entries(indexes)) {
-      if (
-        isRedirect &&
-        new RegExp(`\\${value}(?!\\d)`).test(pathname + (hash || ''))
```

Comment:
We still need this regex

Reply:
Still need it for what? We no longer want to add path segments to the query at all unless they're manually specified by the user which is handled above this loop.

Reply:
If it's not a redirect?

Reply:
If it's a rewrite we pass all path segments through like before

Reply:
```
"rewrite": [
  { "source": "/:path", "destination": "/test?one=:path" }
]
```
Would `/two` still rewrite to `/test?one=two`

Reply:
```
{ source: '/:path', destination: '/test?path=:path' },
```
becomes
```
{
  src: '^(?:\\/([^\\/]+?))$',
  dest: '/test?path=$1',
  check: true,
},
```
I added a test to show this behavior

Reply:
What about `{ source: '/:path/:two', destination: '/test?path=:path' }`

Reply:
```
{ source: '/:path/:two', destination: '/test?path=:path' },
```
becomes
```
{
  check: true,
  dest: '/test?path=$1&two=$2',
  src: '^(?:\\/([^\\/]?))(?:\\/([^\\/]?))$',
},
```
added another test to reflect this also

Reply:
Ok that's good